Source/Csla/Server/DataPortal.cs

```diff
@@ -195,7 +177,7 @@ private Reflection.ServiceProviderMethodCaller ServiceProviderMethodCaller
           error = ex;
         var fex = DataPortal.NewDataPortalException(
             ApplicationContext, "DataPortal.Create " + Resources.FailedOnServer,
-            new DataPortalExceptionHandler().InspectException(objectType, criteria, "DataPortal.Create", error),
+            new DataPortalExceptionHandler(ExceptionInspector).InspectException(objectType, criteria, "DataPortal.Create", error),
```

Comment:
I wonder if DataPortalExceptionHandler should be injected, and pick up the exception inspector automatically? However, that does mean we are not using a new instance on each and every call, so the lifetime is different.

I say this only because it would simplify the code a little - but then making the NewDataPortalException method an instance method could also be used to simplify it.